edkrepo_installer/EdkRepoInstaller/InstallerStrings.cs

```diff
@@ -86,7 +86,7 @@ public static string EdkrepoPython3LauncherScript
         {
             get
             {
-                return "python3";
+                return "python -3.8";

```

Comment:
This is not the command to run the Python interpreter. It is the file name for the shell script that launches the Python interpreter. This file name must continue to be "python3". Otherwise, "#!/usr/bin/env python3" will no longer work.